### DISCUSSION TITLE
Surplus production vs. biomass plot

### DISCUSSION THREAD

**Discussion by akatan999** [nmfs-stock-synthesis/stock-synthesis#565] [General]
Following recent paper (https://cdnsciencepub.com/doi/pdf/10.1139/cjfas-2022-0168), it would be better to plot Surplus production vs. spawining stock biomass instead of total biomass. Or at least add it.

**Comment by e-perl-NOAA** [comment]
@akatan999 Is there anything that needs to be added in ss3 or is it in r4ss that you would like to see this type of plot added?

**Reply by akatan999** [reply]
Only in r4ss. Cheers
